iceoryx_posh/source/roudi/memory/default_roudi_memory.cpp

```diff
@@ -40,27 +41,18 @@ DefaultRouDiMemory::DefaultRouDiMemory(const RouDiConfig_t& roudiConfig) noexcep
 }
 mepoo::MePooConfig DefaultRouDiMemory::introspectionMemPoolConfig() const
 {
+    constexpr uint32_t ALIGNMENT{mepoo::MemPool::CHUNK_MEMORY_ALIGNMENT};
     mepoo::MePooConfig mempoolConfig;
     mempoolConfig.m_mempoolConfig.push_back(
-        {static_cast<uint32_t>(cxx::align(static_cast<uint64_t>(sizeof(roudi::MemPoolIntrospectionInfoContainer)),
-                                          SHARED_MEMORY_ALIGNMENT)),
-         10});
+        {cxx::align(static_cast<uint32_t>(sizeof(roudi::MemPoolIntrospectionInfoContainer)), ALIGNMENT), 10});
```

Comment:
no deal breaker - What does the parameter `10` stand for? Could you please create a constexpr that makes it clear? And do you know why it is `10` and not `5` or `20`? If so could you please add the reasoning with an additional comment.